apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -64,6 +64,8 @@ const Courses = React.createClass({
     if (!this.props.isTeacher) {
       $('#section-management').appendTo(ReactDOM.findDOMNode(this.refs.sectionManagement)).show();
     }
+
+    $('#flashes').appendTo(ReactDOM.findDOMNode(this.refs.flashes)).show();
```

Comment:
Can this be set via Redux instead of using jQuery to manipulate the React component?

+cc @Bjvanminnen